operating/view-dashboard.adoc

```diff
@@ -6,58 +7,100 @@ include::modules/common-attributes.adoc[]
+
+[id="images-at-most-risk"]
+== Images at most risk
+
+The *Images at most risk* widget provides a list of images that have been determined to have the highest risk. ##Risk is determined by an analysis that weighs the risk priority, number of critical common vulnerabilities and exposures (CVEs), and number of important CVEs.##
+
+You can configure which items are displayed in the widget by selecting the following options:
+
+* *Image vulnerabilities*:
+**  *Fixable CVEs*: Displays the number of fixable CVEs in the *Critical CVEs* and *Important CVEs* columns.
+**  *All CVEs*: Displays the total number of CVEs in the *Critical CVEs* and *Important CVEs* columns.
+* *Image status*: 
+** *Active images*: Displays the active images at most risk. 
+** *All images*: Displays all images at most risk.
 
-[id="violations-by-cluster"]
-== Violations by Cluster
+You can also perform the following actions in this widget:
 
-The *Violations by Cluster* widget shows a bar chart based on the number of violations in each cluster in different colors based on their severity.
+* To view CVEs for an image, click on the image name.
+* To view a list of all riskiest images, select *View all*.
 
-[id="top-risky-deployments"]
-== Top Risky Deployments
+[id="deployments-at-most-risk"]
+== Deployments at most risk
 
-The *Top Risky Deployments* widget displays the top five deployments at risk in your containerized environment.
+The *Deployments at most risk* widget provides a list of deployments that have been determined to have the highest risk, ##as determined by the risk priority.## The widget displays the deployment name, cluster and namespace, and risk priority. 
 
-You can select a deployment in this widget to get more details about the issues with that particular deployment.
-You can also select *View All* to view risks for all your deployments.
+You can perform the following actions in this widget:
 
-From here, you can select the separate deployments to see the details of the issues found with each particular deployment.
+* To view the *Risk* page for a deployment, click the deployment name. 
+* To view the *Network Graph* page for the deployment, click the cluster/namespace. 
+* To view all deployments at risk, select *View all*.
 
-[id="active-violations-by-time"]
-== Active Violations by Time
+[id="aging-images"]
+== Active aging images
 
-The *Active Violations by Time* widget shows the scale of number of violations by recent days, with the colors indicating the severity of those violations.
+The Active aging images widget shows the active images by age, allowing you to determine the number of images of each severity level that have not been updated for a selected period of time. 
```

Comment:
Hmm the phrase `of each severity level` kind of makes sense here, but I'm not sure it is something we ever discussed during meetings. Our display of severity for image age is somewhat arbitrary and is not based on a specific measurement like the severity of violations or CVEs; it is purely based on the numbers the user enters for the options. 

The colors are the same as the severity colors in other widgets, but they are mostly a way to distinguish which images are "more old". :)